src/core/backbone_router/ndproxy_table.cpp

```diff
@@ -225,6 +234,77 @@ NdProxyTable::NdProxy *NdProxyTable::FindInvalid(void)
     return found;
 }
 
+void NdProxyTable::HandleTimer(void)
+{
+    VerifyOrExit(mIsAnyDadInProcess);
+
+    {
+        bool isAnyDadInProcess = false;
```

Comment:
Can we consider directly updating `mIsAnyDadInProcess` here in the code below? (remove local `isAnyDadInProcess`)?

Would also help remove the extra `{ }` indentation.